### PR TITLE
Newrelic Addon documentation

### PR DESCRIPTION
**Pull request by bpschmitt**
*Description of changes:*

Adding documentation for the New Relic Addon.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by youngjeong46 — COMMENTED** [review]
Left a few minor comments.

File: docs/addons/newrelic.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  "nrLicenseKey": "xxxxNRAL",
+  "pixieDeployKey": "px-dep-xxxx",
+  "pixieApiKey": "px-api-xxxx"
+}
+```
+
+## Using keys stored in Secrets Manager:
+
+```
+import * as cdk from 'aws-cdk-lib';
```
[the remark this answers is not in the document]
Comment (bpschmitt) [reply docs/addons/newrelic.md:35]: Updated.

File: docs/addons/newrelic.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        installPixie: true,
+        installPixieIntegration: true,
+    }))
+    .region(process.env.AWS_REGION)
+    .account(process.env.AWS_ACCOUNT)
+    .build(app, 'demo-cluster');
+```
+
+## Using keys
+
```
[the remark this answers is not in the document]
Comment (bpschmitt) [reply docs/addons/newrelic.md:59]: Updated.

File: docs/addons/newrelic.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+{
+  "nrLicenseKey": "xxxxNRAL",
+  "pixieDeployKey": "px-dep-xxxx",
+  "pixieApiKey": "px-api-xxxx"
+}
+```
+
+## Using keys stored in Secrets Manager:
+
+```
```
[the remark this answers is not in the document]
Comment (bpschmitt) [reply docs/addons/newrelic.md]: Updated.

**Review by youngjeong46 — APPROVED** [review]
LGTM.

**Review by askulkarni2 — APPROVED** [review]
LGTM!

**Review by shapirov103 — APPROVED** [review]
LGTM and great work!
I will add this page to the nav pane in another PR.
If anything needs adjustment, please submit another, I will fast track it.

**Pull request merged**